### PR TITLE
vmi init fix

### PR DESCRIPTION
**Pull request by nystilaps**
Fix with retry on wrong KPCR register.
Branch is made from commit 34d09bea4ae3647554f064f0a8a8454b9b74a833, that is used for drakvuf.

### CODE REVIEW

**Comment by Wenzel** [comment]
Can you rebase on `libvmi:master` ? it seems that your commit creates a conflict.

**Comment by nystilaps** [comment]
Rebase done.

**Comment by nystilaps** [comment]
I don't understand what is wrong with my code by error messages: 
https://travis-ci.org/libvmi/libvmi/jobs/367124353 
Could you please explain what is wrong or how to fix it?

**Comment by Wenzel** [comment]
@nystilaps i ran the build myself supervised by `scan-build` (clang static analyzer), and the bug is in `libvmi` code itself (not your PR)
report

**Comment by tklengyel** [comment]
@Wenzel the scan-build test skips the config parser folder so that's not the bug it found in this PR

File: tools/windows-offset-finder/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Rekall will find the right GUID, download the PDBs and extract the right offsets
 ## Requirements
 
 - `LibVMI` C library
-- `libvmi` Python bindings
+- [`libvmi` Python bindings](https://github.com/libvmi/python)
```
Comment (tklengyel) [inline tools/windows-offset-finder/README.md]: This is an unrelated change.

**Comment by tklengyel** [comment]
@libvmi-jenkins This is OK to test.

**Comment by nystilaps** [comment]
How is it going?

**Comment by tklengyel** [comment]
@nystilaps looks like you have a bug that trips Travis, so until that is fixed the PR can't be merged.

**Comment by tklengyel** [comment]
Dead store | Dead initialization | core.c | init_from_rekall_profile | 901 | 1
```
const char *kpcr_registers_to_try_names[] = { "GS_BASE", "SHADOW_GS", "MSR_SHADOW_GS_BASE", "FS_BASE" };
--
  | Value stored to 'kpcr_registers_to_try_names' during its initialization is never read

**Comment by nystilaps** [comment]
Quick fix.
